roles/dosomething.base/tasks/install-common-tools.yml

```diff
@@ -12,3 +12,8 @@
     - screen
     - bind9utils
     - sysstat
+
+- name: Disable AppArmor
+  sudo: yes
+  apt: name=apparmor state=absent
+  notify: restart server
```

Comment:
It should be something like this:
http://stackoverflow.com/a/28070759

But, I'm not sure if this is compatible with vagrant.
I'd recommend to reboot the server manually once. I'll investigate this further and make sure servers will be rebooted rebooted automatically when we need it.